translation/models/translation_services.py

```diff
@@ -8,6 +8,24 @@
 from .translation import TranslatedStep, TranslatedLesson, TranslatedCourse, StepSource, TranslatedStepSource
 
 
+# :param: array of texts
+#  make array with escaped symbols (";", "&", "+")
+def escape_symbols(texts):
```

Comment:
why just this symobols are escaped?  May be https://docs.python.org/3.4/library/re.html#re.escape is more suitable?